app/api/business/validators/evidence_validator.py

```diff
@@ -8,6 +8,7 @@ def __init__(self, data, evidence=None):
         self.evidence = evidence
         if self.evidence:
             self.domain = domain_service.find(id=evidence.domain.id).one_or_none()
```

Comment:
Note to self, this one should use `get_by_name_or_id` to avoid the lazy load of suppliers in the Domain model.